deployments/deployments.json

```diff
@@ -7,7 +7,7 @@
     "Arbitrum_SendTokensAdapter": { "address": "0xC06A68DF12376271817FcEBfb45Be996B0e1593E", "blockNumber": 16691987 },
     "Boba_Adapter": { "address": "0x33B0Ec794c15D6Cc705818E70d4CaCe7bCfB5Af3", "blockNumber": 14716798 },
     "Ethereum_Adapter": { "address": "0x527E872a5c3f0C7c24Fe33F2593cFB890a285084", "blockNumber": 14704381 },
-    "SpokePool": { "address": "0x4D9079Bb4165aeb4084c526a32695dCfd2F77381", "blockNumber": 14819486 },
+    "SpokePool": { "address": "0x7CE9f4189d0b0b04B834a5BDeDE8E518Ad1d7dC0", "blockNumber": 17069252 },
```

Comment:
QQ: will we need to be mindful of the data from the old spoke pool after a sufficiently large number of periods?

Reply:
We will not have to care about that data anymore, what do you mean?